solidity/test/router.test.ts

```diff
@@ -24,6 +24,12 @@ const destination = 2;
 const destinationWithoutRouter = 3;
 const body = '0xdeadbeef';
 
+interface GasPaymentParams {
+  gasAmount: BigNumberish;
+  gasPayment: BigNumberish;
```

Comment:
do we need the `gas` prefix? not immediately clear what's different between `gasAmount` and `gasPayment`